### DISCUSSION TITLE
Is it possible to use kmenu on a static website?

### DISCUSSION THREAD

**Discussion by tiagozip** [harshhhdev/kmenu#20] [Q&A]
Hi!
Is it possible to use the kmenu library on a static website?
Thanks!
**Update:** This means: _Can I use kmenu without React?_

**Comment by haaarshsingh** [comment]
Hey,

It's possible to use it on a static website as long as that website uses React. This isn't a library which uses server for anything, it's just a command menu component at the core.

**Reply by haaarshsingh** [reply]
In that case, you can't use kmenu.
As the website suggests, it's a 'An accessible and animated command menu for React'. 
It's currently not framework agnostic, though it might be some day.
